### PR TITLE
adding myself

### PR DESCRIPTION
**Pull request by roopali8**
added my key in the files..

### CODE REVIEW

File: hiera/data/secrets/common.yaml [hunk 1]
```diff
@@ -41,6 +41,7 @@ rjil::system::accounts::local_users:
...
+roopali8: {realname: 'Roopali Rawat', sshkeys: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC+IClmvJmdKy2crOJ9hTeY9M+gU/nw7WGHByYgbvxeVu/KTlNeLiRc35cbI2IfbqNen3KF8CiH/jR/I5Kf8Fu0PwxwRy5m7Hlj6dE6wMQHZG9cB0QHkgUOVf+AyUrvBmCULAuIU7Op5hj4asH10FlLWiZRgD7DhxXvz7KqBc1pzH3a47AjeCX1/jqZn9colTwVq3XdrLFgC85tOrxPsUxUlYdHx+FvaFOJdZHQvzmf2YBnPIEN7M4QC+YKOE9npcfTQNmrWhI0uyh1atX4kTOVlveeT9r4IU05LnJ2Vz+5HTHDRQUUMsSZSkBGz5nvPiHLaaSNXkZ7ZezWC4TKloA+3RfjQT+/FZ6BpFuYE1Sy4+2mKUDqYPMWRdqQ7m4SulfE4yz00E6JEljnJ18X31nly3ct92NTAj3+3VfmX3VulGZd5he3nCY2n9oKQcksALPxdMq9ILbovLn54YUvaI62D/cQF5WlrcyxZ8SWAKkqDtBLWTpfJgWRxdeiXGccjgpBnbpGJJ9PHqv1BAAdA8Ws+SBApnT2Dq53iG2M0/pjHy/MGiztnDaUupmSvmvdzFG9M1qRoPEYONWTKmiFEPxkQK7OpSfhzJg7wPpRRtMfAX5ZUvPHS4ga44kt6tNnondjO5PHpG/lW4SvnZuoBVDLqupcaP396sZ8dyFHZMRksw== roopali8.rawat@gmail.com", password: '*'}
```
Comment (mayankkapoor) [inline hiera/data/secrets/common.yaml]: The indentation doesn't look right here. Better to have in line with above lines (e.g. bhavin).

Comment (anshprat) [inline hiera/data/secrets/common.yaml]: Correct. Please note this is yaml file and indentation is a part of syntax.

**Comment by anshprat** [comment]
Also, this will need a rebase now.

**Comment by anshprat** [comment]
merged as 846, closing this

**Pull request closed without merging**